### PR TITLE
refact(global): some improvements

### PR DESCRIPTION
**Pull request by rimiti**
## PURPOSE

- display jest coverage
- main function refactored
- jsdoc created

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,11 +4,11 @@
   "main": "index.js",
   "repository": "https://github.com/zeke/antipodes",
   "devDependencies": {
-    "jest": "^26.2.2",
-    "standard": "^14.3.4"
+    "jest": "26.2.2",
+    "standard": "14.3.4"
```
Comment (zeke) [inline package.json]:
```suggestion
    "jest": "^26.2.2",
    "standard": "^14.3.4"
```

Comment (zeke) [reply package.json]: ☝🏼 I'd prefer to follow npm's semver defaults.

**Review by zeke — COMMENTED** [review]
Looks good! Just one request to revert the change to dependency ranges.

**Comment by rimiti** [comment]
Changes done.

**Comment by zeke** [comment]
Thanks again!

**Pull request merged**